src/extension.ts

```diff
@@ -1131,12 +1135,33 @@ async function setup(context: vscode.ExtensionContext, progress: ProgressHandle)
 
   // List of functions that will be bound commands
   const funs: (keyof ExtensionManager)[] = [
-    'editKits',          'scanForKits',        'selectKit',        'setKitByName',          'cleanConfigure',
-    'configure',         'build',              'setVariant',       'install',               'editCache',
-    'clean',             'cleanRebuild',       'buildWithTarget',  'setDefaultTarget',      'ctest',
-    'stop',              'quickStart',         'launchTargetPath', 'launchTargetDirectory', 'debugTarget',
-    'launchTarget',      'selectLaunchTarget', 'resetState',       'viewLog',               'compileFile',
-    'tasksBuildCommand'
+    'editKits',
+    'scanForKits',
+    'selectKit',
+    'setKitByName',
+    'cleanConfigure',
+    'configure',
+    'build',
+    'setVariant',
+    'install',
+    'editCache',
+    'clean',
+    'cleanRebuild',
+    'buildWithTarget',
+    'setDefaultTarget',
+    'ctest',
+    'stop',
+    'quickStart',
+    'launchTargetPath',
+    'launchTargetDirectory',
+    'buildType',
+    'buildDirectory',
+    'debugTarget',
+    'launchTarget',
+    'selectLaunchTarget',
+    'resetState',
+    'viewLog',
+    'compileFile',
```

Comment:
It looks like `'tasksBuildCommand'` was lost in this reformatting.  Can you add it back?

Reply:
Sorry about that. Should be fixed.